docs/sphinx/source/contributing/style_guide.rst

```diff
@@ -70,6 +70,27 @@ the ``docs/readthedocs.org:pvlib-python`` link within the checks
 status box at the bottom of the pull request.
 
 
+.. _documentation-units:
+
+Parameter names and units
+-------------------------
+
+When specifying parameters and their units, please follow these guidelines:
+
+- Use the recommended parameter name and units by :ref:`nomenclature` where possible.
+- Enclose units in square brackets, e.g. ``[W]``.
```

Comment:
```suggestion
- Enclose units in square brackets after the parameter description, e.g., ``Air temperature. [°C]``.
```

Reply:
`Air temperature. [°C]`
`Air temperature. [°C].`
`Air temperature, [°C].`

Got any preference? I'm good with that one. Just asking cause I've seen docstring reformatting using the latest one by @RDaxini . Not a big deal, I'm trying to be consistent.

Reply:
I prefer the first one and seem to recall that we more or less agreed on that somewhere..

I think we should also discuss how we represent the units for angles, e.g., [degrees] or [°]

Reply:
Perfect. I think the previous discussion was probably on #1421 .